src/test/translation_tests.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <tinyformat.h>
+#include <util/translation.h>
+
+#include <boost/test/unit_test.hpp>
+
+BOOST_AUTO_TEST_SUITE(translation_tests)
+
+BOOST_AUTO_TEST_CASE(translation_namedparams)
+{
+    bilingual_str arg{"original", "translated"};
+    bilingual_str format{"original [%s]", "translated [%s]"};
+    bilingual_str result{strprintf(format, arg)};
```

Comment:
https://github.com/bitcoin/bitcoin/pull/27150/checks?check_run_id=11596638131
```bash
src/test/translation_tests.cpp: Expected 0 argument(s) after format string but found 1 argument(s): strprintf(format, arg)
```

Reply:
re: https://github.com/bitcoin/bitcoin/pull/27150#discussion_r1118857439

Thanks! Should be fixed now